synthtool/gcp/templates/node_library/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)
+
+- [ ] Tests and linter pass
```

Comment:
Why do we have these checklists? Doesn't our CI handle all of these checks?

Reply:
I think the idea was to force people to actually have done those things. The CI does indeed handle them and I would guess people mostly blindly put x's